HabitatSampler_v2.0/HabitatSampler_v02.r

```diff
@@ -35,13 +35,18 @@ dataPath<-"./Data/"
 outPath<-paste(wd,"Data/Results/",sep="")
 
 ##0.3##
-source(paste(inPath,"install_packages.r",sep=""))
-source(paste(inPath,"clip.r",sep=""))
-source(paste(inPath,"outer_procedure.r",sep=""))
-source(paste(inPath,"plot_results.r",sep=""))
-source(paste(inPath,"plot_interactive.r",sep=""))
-
-usePack("rgdal","raster","maptools","spatialEco","randomForest","e1071","devtools","installr","velox","rgeos","leaflet","htmlwidgets")
+install.packages("https://cran.r-project.org/src/contrib/Archive/BH/BH_1.69.0-1.tar.gz", repos=NULL, type="source")
+remotes::install_git(
+    "https://github.com/carstennh/HabitatSampler.git",
+    ref = "master",
+    subdir = "R-package",
+    dependencies = NA,
+    upgrade=FALSE,
+    build = TRUE,
+    build_manual = TRUE,
+    build_vignettes = TRUE
+)
+HaSa:::usePack("rgdal","raster","maptools","spatialEco","randomForest","e1071","devtools","installr","velox","rgeos","leaflet","htmlwidgets")
 rasterOptions(tmpdir="./RasterTmp/")
```

Comment:
If we now, install the HaSa package in this script, then we do not need the usePack function. An R package should automatically install dependencies...

Reply:
We also need to add the new arguments to multi_Class_Sampling like RGB... also use HaSa::multi_Class_Samling

Reply:
**First comment:**
The `usePack` function is checking more packages then the ones listed in the `DESCRIPTION` file:
```
Imports:
    sf (<= 0.8-1),
    sp (<= 1.4-1),
    rgdal (<= 1.4-8),
    raster,
    rgeos,
    spatialEco,
    randomForest,
    e1071,
    velox,
    leaflet,
    leafem,
    htmlwidgets
```
We could converge the lists and make sure all dependencies are listed in the `DESCRIPTION` file. Then we could remove `usePack`. 

**Second comment**:
The parameter `RBG` is not mandatory since it has a default value. But you are right, it will help the user to know that it exists. I will add it.

Reply:
**First comment:**
Covered by 932c32a, d84dbf8 and f700f03.

**Second comment:**
Covered by 4929d35

Reply:
yes, all packages in usePack should be in the package DESCRIPTION file